app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import os
+
+
+def _check_path(new_path: str) -> None:
+    cur_path = ""
+    for directory in new_path.split("/"):
+        cur_path = os.path.join(cur_path, directory)
+        if not os.path.exists(cur_path):
+            os.mkdir(cur_path)
+
```

Comment:
you can remake this check, try to use os.makedirs() and remove for loop

Reply:
tried to make this as a check for existing directories in cases of multiple depths. replaced it with os.makedirs()